exercises/kindergarten-garden/canonical-data.json

```diff
@@ -0,0 +1,126 @@
+{
+  "exercise": "kindergarten-garden",
+  "version": "1.0.0",
+  "comments": [
+    "Garden can be created with just a diagram or a diagram and students.",
+    "The default students are:",
+    "  Alice, Bob, Charlie, David, Eve, Fred, Ginny, Harriet, Ileana, Joseph, Kincaid, and Larry."
+  ],
+  "cases": [
+    {
+      "description": "partial garden tests",
+      "cases": [
+        {
+          "description": "test Alice's garden",
```

Comment:
That definitely makes sense so it would explain the reasoning behind the test rather than just what it's doing. I'll fix that along with the similar ones below.